AmplifyPlugins/Auth/Tests/AWSCognitoAuthPluginUnitTests/TaskTests/HostedUITests/AWSAuthHostedUISignInTests.swift

```diff
@@ -229,6 +229,22 @@ class AWSAuthHostedUISignInTests: XCTestCase {
         await fulfillment(of: [expectation], timeout: networkTimeout)
     }
 
+    func testUnableToStartASWebAuthenticationSession() async {
+        mockHostedUIResult = .failure(.unableToStartASWebAuthenticationSession)
+        let expectation  = expectation(description: "SignIn operation should complete")
+        do {
+            _ = try await plugin.signInWithWebUI(presentationAnchor: ASPresentationAnchor(), options: nil)
+            XCTFail("Should not succeed")
+        } catch {
+            guard case AuthError.service = error else {
```

Comment:
nit: can we add an assert for `.unableToStartASWebAuthenticationSession` failure type?